database_refresh/02-cronjob.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: db-refresh-job
+spec:
+  schedule: "30 10 * * 0"
+  concurrencyPolicy: "Forbid"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      activeDeadlineSeconds: 1200
+      template:
+        spec:
+          securityContext:
+            runAsUser: 999
+          containers:
+          - name: dbrefresh
+            image: "postgres:14.1"
+            command:
+            - /bin/entrypoint.sh
+            volumeMounts:
+            - name: db-refresh-script
+              mountPath: /bin/entrypoint.sh
+              readOnly: true
+              subPath: entrypoint.sh
+            env:
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: prisons-rds-instance-output
+                  key: database_name
+            - name: DB_USER
+              valueFrom:
```

Comment:
There are multiple dbs in the hmpps-registers namespace.  the prisons-rds-instance-output is the prison register db